task_6/task_5/Task2.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body style="margin: 10px 381px;">
+    <table style="border-collapse: collapse; width: 100%;">
+<tr style="border-top: 1px solid #b5cde7; 
+           border-left: 1px solid #b5cde7; border-right: 1px solid #b5cde7; color: #3966b3;">
+    <td style="width: 100px; padding: 10px;"> <strong>Company</strong></td>
+    <td style="text-align: center;"> <strong>Q1</strong></td>
```

Comment:
Please use css propertyes instead of strong elements

Reply:
Done

Reply:
You still use <strong> everywhere

Reply:
Done